internal/acl/get.go

```diff
@@ -34,24 +35,44 @@ var GetCmd = &cobra.Command{
 mim acl get <name>
 `,
 	Run: func(cmd *cobra.Command, args []string) {
-		pterm.DisableOutput()
+		format := utils.ResolveFormat(cmd)
+		if format == "json" {
+			pterm.DisableOutput()
+		}
 		server, token, err := login.ResolveCredentials()
 		utils.HandleError(err)
 
-		name := args[0]
+		clientId, err := cmd.Flags().GetString("clientId")
+		utils.HandleError(err)
+		if len(args) > 0 && clientId == "" {
+			clientId = args[0]
+		}
+
+		if clientId == "" {
+			pterm.Warning.Println("You must provide a clientId")
+			pterm.Println()
+			os.Exit(1)
+		}
```

Comment:
this is fine, just wanted to highlight different way: cobra also has a way to "require" a parameter.
It should be possible to replace the if check with 
`GetCmd.MarkFlagRequired("clientId")` in the init function after line 77.

Reply:
Yes, I tried that feature. The reason for not using it is that we currently allow the clientId flag to also be a positional argument without declaring the flag. If we use the MarkFlagRequired feature, the following will cause an error: 

```
mim acl get example-client
```